regexes/bots.yml

```diff
@@ -180,6 +180,14 @@
     name: ''
     url: ''
 
+- regex: 'Better Uptime Bot'
```

Comment:
Hi, thanks for the PR

add test for UA 
```
Better Uptime Bot Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36
```

to quickly create a test fixture, you can use the tool from the `misc` directory
```bash
cd device-detector/misc
php test.php "Better Uptime Bot Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36"
```
+ copy result to buffer
+ insert result to file fixtures device-detector/Tests/fixtures/bots.yml